cdap-ui/app/cdap/components/PlusButtonModal/PlusButtonModal.less

```diff
@@ -53,6 +53,16 @@
           margin-right: 2px;
         }
 
+        .modal-close-btn {
+          width: 51px;
+          height: 100%;
```

Comment:
Neither `height` nor `margin-right`  is needed here. `vertical-align: middle` is what needed here.